### PR TITLE
TestNg-plugin upgrade

### PR DESCRIPTION
**Pull request by prashelke**

### CODE REVIEW

File: GingerTestNgPluginConsole/Ginger.PluginPackage.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "Id": "TestNG",
-  "Version": "v1.1.0",
+  "Version": "v2.0.1",
```
Comment (Maheshkale447) [inline GingerTestNgPluginConsole/Ginger.PluginPackage.json]: change it to v2.0.0

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
   "ExpandedNodes": [
     ""
   ],
+  "SelectedNode": "\\Ginger-TestNG-Plugin.sln",
```
Comment (Maheshkale447) [inline .vs/VSWorkspaceState.json:5]: remove unwanted files

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: .vs/VSWorkspaceState.json (hunk 2 above)
Comment (prashelke) [reply .vs/VSWorkspaceState.json:5]: Done

File: GingerTestNgPluginConsole/Ginger.PluginPackage.json (hunk 1 above)
Comment (prashelke) [reply GingerTestNgPluginConsole/Ginger.PluginPackage.json]: Done

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**